assets/scss/learndash-styling.scss

```diff
@@ -79,7 +79,7 @@
       -ms-flex: initial;
 
       &:hover {
-        color: $color_nhsuk-yellow !important;
+        color: $nhsuk-link-hover-color !important;
```

Comment:
Does this really need an !important on it?

Reply:
unfortunately so. The learndash css is also !important and outside our control so our code needs an !important to take effect.